config/deploy/prod_server.rb.template

```diff
@@ -1,44 +0,0 @@
-set :application, "nucore"
```

Comment:
Mainly it's supplanted by `deploy.rb` and the individual deploy environments. Ultimately, this is a much older capistrano setup, and the dsl has become much simpler these days.